rose/client/main.py

```diff
@@ -101,7 +101,7 @@ def main():
     try:
         driver_mod = load_driver_module(sys.argv[1])
     except ImportError as e:
-        print "error loading driver module %r: %s" % (sys.argv[1], e)
+        log.warning("error loading driver module %r: %s" % (sys.argv[1], e))
```

Comment:
We want to pass the arguments to the logger instead of formatting them before calling the logger. Also, this is an error, should use log.error().